nixos/modules/services/networking/zerotierone.nix

```
@@ -97,7 +118,11 @@ in
     # ZeroTier receives UDP transmissions
     networking.firewall.allowedUDPPorts = [ cfg.port ];
 
-    environment.systemPackages = [ cfg.package ];
+    environment.systemPackages =
+      [ cfg.package ]
+      ++ lib.optional (
+        cfg.generateSystemdNetworkdConfig && cfg.systemdManagerPackage != null
```

lib.mkPackageOption is non-nullable by default.

```suggestion
        cfg.generateSystemdNetworkdConfig
```